frontend/src/concepts/analyticsTracking/useTrackUser.ts

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useUser } from '~/redux/selectors';
+import { useAccessReview } from '~/api';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import { IdentifyEventProperties } from '~/concepts/analyticsTracking/trackingProperties';
+
+export const useTrackUser = (username?: string): [IdentifyEventProperties, boolean] => {
+  const { isAdmin } = useUser();
+  const [anonymousId, setAnonymousId] = React.useState<string | undefined>(undefined);
+
+  const [loaded, setLoaded] = React.useState(false);
```

Comment:
Unnecessary to track this `loaded` as a separate state. 
Compute it as `!!anonymousId && acLoaded`